cves/kernel/CVE-2021-38204.yml

```diff
@@ -129,10 +128,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    No automated tests were found in the surround directories and there is no reference to tests in the commit message.
+  fix: false
+  fix_answer: |
```

Comment:
It's good coding practice to have a reasonable line length.

Reply:
updated line lengths on some lines